contents/blog/startup-tigris.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Startup spotlight: Tigris Data"
+date: 2023-04-20
+author: ["charles-cook"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/transparency.jpeg
+featuredImageType: full
+category: Startups
+---
+
+This month we're chatting to Tigris co-founder & CEO Ovais Tariq. Prior to starting Tigris, Ovais spent several years at Uber where he was Head of Storage Infra.
+
+> Every month we shine the spotlight on a different startup to show off the cool things they're building, as part of the PostHog for Startups program. Want a chance to be featured? [Sign up here for $50k in PostHog credit and a bunch of other sweet perks!](/startups) 
+
+### What is Tigris and why is it cool?
+
+"[Tigris](https://www.tigrisdata.com/) is a developer data platform that provides an integrated suite of data services for application developers. These services include a NoSQL database for storing structured and unstructured data, and an integrated search engine that allows users to do both full-text and vector searches. We've got more services coming soon, including event streaming and caching.
+
+We chose these services because most real-time applications require a combination of them. If you think of an e-commerce application, you need a way to store the product data, store the user data, store the orders-management data, a way to search everything etc. Usually you need a combination of different types of products to do this. 
+
+What we are trying to do is, instead of having the developer deal with all these solutions and having them deal with connecting them together and having a complex infrastructure, we're providing them with everything integrated. That means they can focus on just writing the application code and business logic, and we take care of all the infrastructure related to data storage and access."
+
+### How did you decide this was a problem worth solving?
+
+"I've been in the data infra space for 17 years. Before starting Tigris, I spent around 6 years at Uber where I was leading the teams that built Uber's internal database platform. At Uber it was very clear that a lot of developer time was spent dealing with different types of solutions and connecting them together. So we ended up dealing with operational infrastructure instead of focusing on writing features fast enough. We built a solution called Docstore which integrated many different data-access platforms together behind a single API. And that allowed us to move really fast and build new types of use cases. 
```

Comment:
```suggestion
"I've been in the data infra space for 17 years. Before starting Tigris, I spent around 6 years at Uber, where I was leading the teams that built Uber's internal database platform. At Uber, it was very clear that a lot of developer time was spent dealing with different types of database solutions and connecting them together. So developers would end up dealing with operational infrastructure instead of focusing on writing features fast enough. We built a solution called Docstore which was a general-purpose database platform supporting wide variety of workloads and data access patterns through a single unified API. And that allowed developers to move really fast and build new types of use cases. 
```